Tools/clinic/libclinic/dsl_parser.py

```diff
@@ -423,6 +425,13 @@ def at_critical_section(self, *args: str) -> None:
         self.target_critical_section.extend(args)
         self.critical_section = True
 
+    def at_disable(self, *args: str) -> None:
+        if self.kind is not CALLABLE:
+            fail("Can't set @disable, function is not a normal callable")
+        if not args:
+            fail("@disable expects at least one argument")
+        self.disable_fastcall = 'fastcall' in args
```

Comment:
I would prefer stricter parsing, something like:

```suggestion
        if args == 'fastcall':
            self.disable_fastcall = True
        else:
            self.fail("Invalid @disable option")
```

Reply:
Since `args` is a list of arguments, I don't think I'm able to do it that way. Directives are `@NAME arg1 arg2 ...`. So what I can do is however check that `args` has length 1 only.